packages/angular/build/src/builders/dev-server/vite-server.ts

```diff
@@ -841,6 +841,8 @@ export async function setupServer(
           ? (proxy ?? {})
           : proxy,
       cors: {
+        // This will include `Access-Control-Allow-Origin: http://{{Origin}}`
```

Comment:
Will the header sticked then to `http://`? Or should the comment more looks like this:

```suggestion
        // This will include `Access-Control-Allow-Origin: {{Origin}}`
```